libs/beacon/tests/dkg/full_cycle.cpp

```diff
@@ -126,6 +114,18 @@ struct CabinetNode
   {
     return fetch::network::Uri{"tcp://127.0.0.1:" + std::to_string(muddle_port)};
   }
+
+  static ProverPtr CreateNewCertificate()
```

Comment:
Hrm, we should probably put these in some sort of convenience header since there is 100s of uses of it

Reply:
Yes, I agree.